docs/pages/getting-started/adapters/drizzle.mdx

```diff
@@ -99,11 +99,12 @@ import {
   integer,
 } from "drizzle-orm/pg-core"
 import type { AdapterAccount } from "next-auth/adapters"
+import { randomUUID } from "crypto"
 
 export const users = pgTable("user", {
-  id: text("id").notNull().primaryKey(),
+  id: text("id").primaryKey().$defaultFn(() => randomUUID()),
```

Comment:
Be aware that currently this won't have any effect.

See: https://github.com/nextauthjs/next-auth/pull/10479#discussion_r1573283226

Reply:
thanks for the heads up. Why will this not have any effect? Because the UUID is generated already somewhere else (I had a hard time following the detailed conversation in the link you shared...)

Also: If there's a bit refactoring of the adapter going on, should this PR maybe be delayed or even deleted?